packages/server/pm2.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   apps: [
     {
-      script: "dist/index.js",
+      script: "dist/index.ts",
```

Comment:
Is this right?

Reply:
No it isn't thanks for catching that, was an errant re-name!